src/extension/sldService/src/test/java/org/geoserver/sldservice/rest/RasterizerTest.java

```diff
@@ -68,6 +68,7 @@ public void testRasterizeWithNoParams() throws Exception {
         final String restPath = RestBaseController.ROOT_PATH + "/sldservice/wcs:World/" + getServiceUrl() + ".xml";
         MockHttpServletResponse response = getAsServletResponse(restPath);
         assertEquals(200, response.getStatus());
+        login(); // re-login, AdminRequest's gone
```

Comment:
Interesting... I believe there are plenty of multi-request REST tests out there, but maybe this was actually one of the few to leverage the AdminRequest?

Some random examples of multi-request tests:

* CoverageControllerTest.testPutWithCalculation
* CoverageStoreControllerTest.testPutEmptyAndHarvest
* DataStoreController.getWrongDataStore
* LayerControllerTest.testPutWorkspaceStyle

and so on, there is actually a lot.